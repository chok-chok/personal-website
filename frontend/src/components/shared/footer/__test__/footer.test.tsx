import * as React from 'react';
import Footer from '../footer';
import { shallow } from 'enzyme';

describe('Test Footer component', () => {
  it('should renders correctly', () => {
    const tree = shallow(<Footer />);
    expect(tree).toMatchSnapshot();
  });
});
