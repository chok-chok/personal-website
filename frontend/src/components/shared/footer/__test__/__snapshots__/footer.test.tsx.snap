// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer component should renders correctly 1`] = `
<styled__StyledFooter>
  <SocialProfile />
</styled__StyledFooter>
`;
